sdi/helper.go

```diff
@@ -0,0 +1,71 @@
+package sdi
+
+import (
+	"bytes"
+	"encoding/xml"
+	"fmt"
+	"mime"
+	"mime/multipart"
+	"strings"
+
+	resty "github.com/go-resty/resty/v2"
+)
+
+// ExtractXMLContent extracts the content inside specified XML tag.
+func ExtractXMLContent(data string, tag string) (string, error) {
```

Comment:
Do these methods need to be public?

Reply:
I changed it to a private one, but I had to move it to the `sdi_test` package.